Source/WebKit/WebProcess/WebCoreSupport/WebChromeClient.cpp

```diff
@@ -1042,9 +1042,7 @@ RefPtr<WebCore::WebGPU::GPU> WebChromeClient::createGPUForWebGPU() const
 #if ENABLE(GPU_PROCESS)
     return RemoteGPUProxy::create(WebProcess::singleton().ensureGPUProcessConnection().connection(), WebGPU::DowncastConvertToBackingContext::create(), WebGPUIdentifier::generate(), protectedPage()->ensureRemoteRenderingBackendProxy().ensureBackendCreated());
 #else
-    return WebCore::WebGPU::create([](WebCore::WebGPU::WorkItem&& workItem) {
-        callOnMainRunLoop(WTFMove(workItem));
-    });
+    return nullptr;
```

Comment:
Does this remove support for running WebGPU in WK1/WebProcess?

If it previously somewhat worked (at least for testing), then it'd be good to have this mentioned in the commit message (even in the title, if someone might have been using it).

Reply:
We haven’t supported WebGPU in WK1 for over a year and we don’t have plans to add support for it

Reply:
😭

Reply:
oh looks like @achristensen07 wrote some tests which use WebGPU and WK1. I will put this back but generally it is not supported